packages/commonwealth/server/routes/threads/getThreads.ts

```diff
@@ -74,7 +74,10 @@ export const getThreads = async (
   if (topic_id) where['topic_id'] = topic_id;
   if (address_ids) where['address_id'] = { [Op.in]: address_ids };
   if (include_comments)
-    include.push({ model: models.Comment, required: false });
+    include.push({
+      model: models.Comment.scope('excludeAttributes'),
```

Comment:
only comment title is included, this actually can break system - if not removed!
As getComments is paged - thats much more safer
- We cannot include all comments associated with thread - just their titles - as we have 100s of comments

Reply:
Not sure if any bots are using getThreads to fetch all comments at once! 
for react app getThreads - include_comments is always false

We have to test seprately if we like to include body etc! include_comments=true!

Reply:
The include_comments flag is a problem anyway. I guess we're lucky it didn't already break.

Reply:
Should cc @zakhap here

Reply:
What we can do is 

one of these, as these feed are read only

text - some query string format
Plaintext - readable good for external api
Body - markdown
Version history

Reply:
For now - changed it to 
- models.Comment.scope('readOnly')
- basically includes text, body, plaintext

We should deprecate `include_comments` or change meaning to just include ids and one of plaintext & text